### PR TITLE
fix Id constant change

### PR DESCRIPTION
**Pull request by mcalhoun**
## what
* Fix a bug where the topic of the SNS Policy was constantly changing on each plan/apply cycle.

## why
* This is a bug that was occurring because we were creating duplicate overlapping SNS Topic Policies. This PR fixes that issue.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -29,6 +29,8 @@ module "sns_topic" {
   subscribers     = var.subscribers
   sqs_dlq_enabled = false
 
+  allowed_aws_services_for_sns_published = ["cloudwatch.amazonaws.com"]
```
Comment (korenyoni) [inline main.tf:32]: Btw, this is actually the default for this module https://github.com/cloudposse/terraform-aws-sns-topic/blob/master/variables.tf#L19 — not sure if you wanted to keep this in just so it's more declarative

Comment (mcalhoun) [reply main.tf:32]: Yep, I just wanted to be explicit/clear when looking at this code :)

Comment (korenyoni) [reply main.tf:32]: This module could be also bumped to latest (`0.16.0`)

Comment (mcalhoun) [reply main.tf:32]: Ah...good call I'll do that now.

**Review by korenyoni — DISMISSED** [review]
Looks good to me.

**Review by korenyoni — DISMISSED** [review]
Re-approving after `sns-topic` module version bump

File: main.tf (hunk 1 above)
Comment (mcalhoun) [reply main.tf:32]: 4bda0e04eeb3a58bfbe7502bef15477327afbead

**Review by korenyoni — APPROVED** [review]
Looks good to me 👍

**Pull request merged**